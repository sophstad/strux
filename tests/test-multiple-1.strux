int main() {
	int i = 3;
	num j = 9.0;
	print(j*j);
	print(i*i);
	return 0;
}
