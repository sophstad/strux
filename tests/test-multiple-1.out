81.000000
9
